src/Svg/Tag/Text.php

```diff
@@ -16,18 +18,18 @@ class Text extends Shape
 
     public function start($attributes)
     {
-        $document = $this->document;
```

Comment:
There was not anything for dx/dy yet so I guess adding logic to support it plus percentages is asking too much. There are already some text-related issues to address so we can push off further work on this element to a future release.

Reply:
Okay, Have not applied any further changes.